src/SlackConnector/Connections/Clients/Chat/FlurlChatClient.cs

```diff
@@ -13,13 +13,14 @@ internal class FlurlChatClient : IChatClient
     {
         private readonly IResponseVerifier _responseVerifier;
         internal const string SEND_MESSAGE_PATH = "/api/chat.postMessage";
+        internal const string DELETE_MESSAGE_PATH = "/api/chat.delete";
 
         public FlurlChatClient(IResponseVerifier responseVerifier)
         {
             _responseVerifier = responseVerifier;
         }
 
-        public async Task PostMessage(string slackKey, string channel, string text, IList<SlackAttachment> attachments)
+        public async Task<PostMessageResponse> PostMessage(string slackKey, string channel, string text, IList<SlackAttachment> attachments)
```

Comment:
Hmm, I haven't SpecsFor before, but it looks like you used it for Say test. However as far as I can see, it doesn't support async calls, so in cases where you need to get the results of an async call, you're not using SpecsFor and just adding comments for given, when, then. For example FlurlChatClientTests.

So basically I'd need to convert the Say Unit test from using SpecsFor to be more like FlurlChatClientTests. Is that correct?